.github/workflows/test_vsf1.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
 
       - name: Clone default theme
-        run: git clone --quiet --single-branch --branch master https://github.com/DivanteLtd/vsf-default.git ./src/themes/default
+        run: git clone --quiet --single-branch --branch master https://github.com/vuestorefront/vsf-default.git ./src/themes/default
```

Comment:
this pr should be merged **after** `vsf-default` is moved to `vuestorefront`

Reply:
done, thx ;)